src/test/resources/security/sample.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Why do we need this file?

Reply:
Its the cert  for the key `plugins.security.ssl.http.pemcert_filepath`